### ISSUE TITLE
Add a cli

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/webidl2.js#598]
Maybe we want a separate project for this (`-cli`) that allows for validation from the command line. 

```
webidl --validate some.webidl
```

This would be great for CI, to allow Bikeshed document's to be checked for errors (the WebIDL parse/processor in Bikeshed is a bit limited).

**Comment by saschanaz** [comment]
Sounds good, but I don't think `.webidl` works for bikeshed? Does it support extracting idls to separate files?

**Comment by marcoscaceres** [comment]
yeah, we would need to extract the IDL by some other means (i.e., this would be spec-writing tool agnostic). But that's fine.

**Comment by dontcallmedom** [comment]
https://github.com/w3c/reffy/blob/master/src/cli/extract-webidl.js is a tool that extract IDL fragments from specs independently of their authoring tool

**Comment by saschanaz** [comment]
https://github.com/saschanaz/webidl-updater/blob/cabc499302214586f84c7186b4497a06793fc977/src/utils/extract-webidl.js is my tweaked version of it for rewriting.

Sounds like it could be useful to have that logic here, probably in a separate package.
